src/components/LandingPage.js

```diff
@@ -2,30 +2,42 @@ import React from 'react';
 import {connect} from 'react-redux';
 import {Hero, Section} from 'react-landing-page';
 import {GoogleLogin} from 'react-google-login';
+import axios from 'axios';
 import Cookies from 'js-cookie'
+import {fetchWithTimeout} from '../util/FetchWithTimeout';
 import {
   BACKEND_MODE_SIGNED_IN_STATUS,
+  SERVER_BASE_URL,
   ACCESS_TOKEN_COOKIE_KEY,
-  loginBackend,
   setBackendModeSignedInStatusAction,
 } from '../reducers/BackendModeSignedInStatus';
 
 const CLIENT_ID = '709358329925-gic89ini15sgaenfrta1gshej1ik72jg.apps.googleusercontent.com';
 
 class LandingPage extends React.Component {
 
-  handleLoginSuccess = response => {
-    if (response.tokenObj) {
-      loginBackend(response.profileObj.name, response.profileObj.email, response.tokenObj.id_token).then(ok => {
-        if (ok) {
-          Cookies.set(
-            ACCESS_TOKEN_COOKIE_KEY,
-            response.tokenObj.id_token,
-            { expires: new Date(response.tokenObj.expires_at) },
-          );
-          this.props.dispatchSetBackendModeSignedInStatusAction(BACKEND_MODE_SIGNED_IN_STATUS.USER_SIGNED_IN);
-        }
-      });
+  handleLoginSuccess = async ({ profileObj: { name, email }, tokenObj: { id_token: token, expiresAt: expiry } }) => {
+    let response = null;
+    try {
+      response = await fetchWithTimeout(
+        SERVER_BASE_URL + 'auth',
+        { method: 'GET', headers: new Headers({ 'Set-Cookie': `token=${token}` }) },
+      );
+      if (!response.ok) {
```

Comment:
What's the point of this `auth` call? We don't need this as `register` will auth also